### PR TITLE
docs(components): Adding Icon to New Docs Site [JOB-110366]

### PR DESCRIPTION
**Pull request by Beccaneer**
### Changed

Added the Icon component to the new docs site.

## Testing

Run the docs site from the packages/site directory with `npm run dev`.
You should see an 'Icon' option after clicking on Components.

Changes can be
[tested via Pre-release](https://github.com/GetJobber/atlantis/actions/workflows/trigger-qa-build.yml)
---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/site/src/content/index.ts [hunk 1]
```diff
@@ -40,13 +41,16 @@ export const SiteContent: Record<string, ContentExport> = {
   Countdown: {
     ...CountdownContent,
   },
+  Disclosure: {
```
Comment (Beccaneer) [inline packages/site/src/content/index.ts:44]: This is changed because I put it in alphabetical order

**Comment by chris-at-jobber** [comment]
Looks good - just some merge conflicts

**Comment by Beccaneer** [comment]
@chris-at-jobber Yeah, the merge conflicts are going to happen every time :(

**Review by scotttjob — APPROVED** [review]
LGTM!

We'll get an eslint step setup to auto-alphabetize object keys.

**Pull request merged**